packages/netlify-cms-backend-github/src/API.ts

```diff
@@ -672,9 +675,14 @@ export default class API {
     const newContentKey = `${metadata.collection}/${oldContentKey}`;
     const newBranchName = `cms/${newContentKey}`;
 
-    // create new branch and pull request in new format
-    await this.createBranch(newBranchName, pullRequest.head.sha as string);
-    const pr = await this.createPR(pullRequest.title, newBranchName);
+    // retrieve or create new branch and pull request in new format
+    const branch = await this.getBranch(newBranchName);
```

Comment:
This will actually throw when the branch doesn't exist.
I'll add a fix. Another way to do it is just try to create the branch and ignore the `reference exists` error.